modules/wizard/wizard.php

```diff
@@ -274,6 +276,10 @@ public function wizard_notice() {
 	 */
 	public function display_wizard_page() {
 		set_current_screen( 'pll-wizard' );
+		do_action( 'admin_enqueue_scripts' );
```

Comment:
Where does this `do_action()` come from?

Reply:
Keep in mind that the wizard is outside a WordPress admin context. So we need to enqueue our scripts ourselves.

Reply:
I understand that, but I can find where it's moved from.

Reply:
It was in `view-wizard-page.php` https://github.com/polylang/polylang/pull/1443/files#diff-3a4ba45b721c33af7e4372a25537b6a042cd2a5aa3c5694a0e2ec26a84b3b623

https://github.com/polylang/polylang/blob/master/modules/wizard/view-wizard-page.php#L35

Reply:
Thanks!
Indeed, way better your way.